probes/entries.go

```diff
@@ -117,6 +120,45 @@ var (
 		issueActivityByProjectMember.Run,
 		notCreatedRecently.Run,
 	}
+
+	probeRunners = map[string]func(*checker.RawResults) ([]finding.Finding, string, error){
+		securityPolicyPresent.Probe:                         securityPolicyPresent.Run,
+		securityPolicyContainsLinks.Probe:                   securityPolicyContainsLinks.Run,
+		securityPolicyContainsVulnerabilityDisclosure.Probe: securityPolicyContainsVulnerabilityDisclosure.Run,
```

Comment:
long term we should find another way of doing this. This is just asking for someone to forget adding an entry later

Reply:
#3720